### PR TITLE
Webui

### PR DESCRIPTION
**Pull request by oderah**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,10 +3,20 @@
...
+    "@types/node": "^12.12.16",
+    "@types/react": "^16.9.16",
+    "@types/react-dom": "^16.9.4",
+    "axios": "^0.19.1",
+    "bootstrap": "^4.4.1",
+    "lodash": "4.17.13",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-scripts": "3.2.0",
-    "serialize-javascript": "^2.1.1"
```
Comment (nnam-ibe) [inline client/package.json]: Did you remove this on purpose?

File: client/src/App.css.map [hunk 2]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["themes/App.scss","themes/_variables.scss"],"names":[],"mappings":"AAEA;EACE;EACA;EACA,OCDkB;;ADkBlB;EACE;;AAGF;EACE,YCLS;;ADWL;EACE,kBC/BS;;ADqCL;EACE,OCnCK;;AD4Cf;EACE,kBC7Ca;;AD+Cb;EACE,OCpDc;EDqDd;;AAGE;EACE,OCzDU;;ADgEpB;EACE,kBCjEkB;;ADoEpB;EACE,kBClEiB;;ADuEnB;EACE;;AAGF;EACE,kBC1Ea;;AD4Eb;EACE;;AAKN;EACE;EACA","file":"App.css"}
```
Comment (nnam-ibe) [inline client/src/App.css.map]: You added `.map` to the gitignore, was it supposed to exclude this file?

File: client/yarn.lock [hunk 3]
```diff
@@ -8602,11 +8983,6 @@ serialize-javascript@^1.7.0:
   resolved "https://registry.yarnpkg.com/serialize-javascript/-/serialize-javascript-1.9.1.tgz#cfc200aef77b600c47da9bb8149c943e798c2fdb"
   integrity sha512-0Vb/54WJ6k5v8sSWN09S0ora+Hnr+cX40r9F170nT+mSkaxltoE/7R3OrIdBSUv1OoiobH1QoWQbCnAO+e8J1A==
 
-serialize-javascript@^2.1.1:
-  version "2.1.2"
-  resolved "https://registry.yarnpkg.com/serialize-javascript/-/serialize-javascript-2.1.2.tgz#ecec53b0e0317bdc95ef76ab7074b7384785fa61"
-  integrity sha512-rs9OggEUF0V4jUSecXazOYsLfu7OGK2qIn3c7IPBiffz32XniEp/TX9Xmc9LQfK2nQ2QKHvZ2oygKUGU0lG4jQ==
-
```
Comment (nnam-ibe) [inline client/yarn.lock]: same as above, serialize-javascript@^2.1.1 removed

File: package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (nnam-ibe) [inline package.json]: Did we need another package.json file?

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,280 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (nnam-ibe) [inline yarn.lock]: And a lockfile

File: client/package.json (hunk 1 above)
Comment (oderah) [reply client/package.json]: Yes, I removed it when I switched to Typescript. Do we need it for anything?

File: client/src/App.css.map (hunk 2 above)
Comment (oderah) [reply client/src/App.css.map]: Yes, I've been trying to exclude that file. It doesn't seem to work.

File: package.json (hunk 4 above)
Comment (oderah) [reply package.json]: I had no idea this existed, I'll remove it.

File: yarn.lock (hunk 5 above)
Comment (oderah) [reply yarn.lock]: I'll remove it as well

File: client/package.json (hunk 1 above)
Comment (nnam-ibe) [reply client/package.json]: I think there was a security vulnerability with some version of the package that a module we use uses @OGBA-B can confirm. 

Upgrading what ever package it is, is probably the way to go, assuming they've fixed the issue by now, so I don't mind removing it

File: client/src/App.css.map (hunk 2 above)
Comment (nnam-ibe) [reply client/src/App.css.map]: Try using `git rm` to untrack the file.

**Pull request merged**